### ISSUE TITLE
[BUG] Getting 'Failed to execute setRequestHeader error' when searching Chinese/Korean on Search Preview panel

### ISSUE THREAD

**Issue by tim5go** [appbaseio/dejaVu#271]
As titled, I am getting the following the error when searching non-english keywords on the **Search Preview** panel. 
```
TypeError: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': Value is not a valid ByteString.
    at vendors~main.8b56a9e2.js:89
    at u.forEach (vendors~main.8b56a9e2.js:89)
    at vendors~main.8b56a9e2.js:89
    at new Promise (<anonymous>)
    at e.fetch (vendors~main.8b56a9e2.js:89)
    at vendors~main.8b56a9e2.js:19
    at new Promise (<anonymous>)
    at m.g [as performFetchRequest] (vendors~main.8b56a9e2.js:19)
    at m.S [as msearch] (vendors~main.8b56a9e2.js:19)
    at vendors~main.8b56a9e2.js:35
```

**Comment by lakhansamani** [comment]
@tim5go are you adding any custom headers here?

**Comment by tim5go** [comment]
@lakhansamani 
Thanks for your reply!
I am using ES `6.5.4`
The only custom headers I added are: 
`http.cors.allow-headers=X-Requested-With,X-Auth-Token,Content-Type,Content-Length,Authorization,x-search-query,x-search-filters`

**Comment by siddharthlatest** [comment]
@tim5go I get the error. ElasticSearch doesn't allow setting custom headers: `x-search-query` and `x-search-filters` are causing these -> which are enabled in search preview for tracking analytics. However, since analytics on search isn't available outside of appbase.io, we should disable this prop when using it outside of appbase.io and Arc settings. cc @metagrover

**Comment by tim5go** [comment]
@siddharthlatest 
I don't quite get it. It's strange that it only happens to non-english keyword search on **Search Preview**. And even after removing the headers `x-search-query` and `x-search-filters`, the error still occurs.

**Comment by metagrover** [comment]
I agree this error is related to non-english keyword encoding, not the extra headers. I think we may have to use `encodeURI()` to get past this.

**Comment by IfritScv** [comment]
I'm also getting this error searching with Arabic language. However, I'm not sure if this is causing the data grid to not refresh upon receiving the response from ES.

In-my case the response is received correctly as JSON but the UI on Search Preview refuses to refresh/updates.

**Comment by mifopen** [comment]
I have the same issue with Russian. Any chance to see it fixed?

**Comment by mifopen** [comment]
Oh, sorry, haven't noticed the project is dead...
